Source/Core/Core/PowerPC/Jit64/Jit_FloatingPoint.cpp

```diff
@@ -307,7 +307,7 @@ void Jit64::fmaddXX(UGeckoInstruction inst)
   // is created on a CPU that has FMA instructions and then gets played back on a CPU that doesn't.
   // (Or if the user just really wants to override the setting and knows how to do so.)
   const bool use_fma = Config::Get(Config::SESSION_USE_FMA);
-  const bool software_fma = use_fma && !cpu_info.bFMA;
+  const bool software_fma = use_fma;
```

Comment:
Uuh I assume this was left in form debugging?

Reply:
Oh oops, yes